src/styles/_base.sass

```diff
@@ -119,8 +119,8 @@ a
 	margin-left: 0px
 
 strong
-	font-weight: 300
-	color: #fff
+	font-weight: 700
+	color: #ddd
```

Comment:
Why not inherit the font color from where the element is being used?

```suggestion
```

Reply:
This was my attempt to adjust the brightness so that the bold text does not "overshine".

Reply:
I guess that works for now